changes/process_changes.py

```diff
@@ -0,0 +1,124 @@
+# coding: utf-8
+
+from __future__ import absolute_import, division, print_function
+
+import argparse as A
+from funcy import memoize
+from glob import glob
+import os
+from os import path as P
+import re
+import yaml
+
+
+TN_RE = re.compile(r"[A-Z]\d\d\d-\d\d\d")
+
+
+def all_entries():
+    return [f for f in glob('*.yaml') if not f == 'entry_schema.yaml']
+
+
+types_to_header = {
+    'api-change': 'breaking changes',
+    'new-feature': 'new features'
+}
+
+
+def create_rst(change_type="api-change"):
+    """
+    Create an RST file from all the change entries for a given ``change_type``.
```

Comment:
Currently, this function does not create an RST file (I mean does not write one to the disk) but rather prints RST content on the standard output. I we keep it that way, I suggest to update this docstring.

Correct me if I’m wrong, but for now the Sphinx doc will contain only breaking changes (not new features). Looks like this wasn’t the intent. What about we change the `create-rst` command to run `create_rst` once per change type? This would generate all sections (for all change types) in a single RST document, which sounds good to me.

Reply:
> Looks like this wasn’t the intent

It was. The origin of all this is that I wanted an easy way for our users (mostly internal), and even for us, to glance at the breaking changes since last time they updated.

> This would generate all sections (for all change types) in a single RST document, which sounds good to me.

I wanted to have a different document/section per change type, to make it easier to browse. What do yo uthink ?

Reply:
Yes sorry this is what I meant to suggest: generate a single RST document that contains one section per change type, and then all change types listed there.

Reply:
OK, done